hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -21,6 +21,9 @@ parameters:
   - name: FEATURE_TOGGLES
```

Comment:
I don't know how this olm-artifacts-template works, but is it correct that `FEATURE_TOGGLES` and `FEATURE_FLAGS` are required here when they are optional in the addon-operator-spec. And also `FEATURE_TOGGLES` isn't mentioned below

Reply:
Yep you're correct, they can be patched to be optional as well

Reply:
I initially went ahead with "Required: true" because with that specification, the saas-herder perform some additional validation to ensure the validity of our saas-addon-operator configuration, therefore, I went ahead with it. 

That being said, required being true or false, functionally, it creates no difference for ADO :)